perun-utils/ldapc-scripts/schemas/perun-schema.ldif

```diff
@@ -63,12 +63,14 @@ olcAttributeTypes: {58}( 1.3.6.1.4.1.8057.2.80.64 NAME 'sponsoredMembershipInOrg
 olcAttributeTypes: {59}( 1.3.6.1.4.1.8057.2.80.65 NAME 'userIdentities' DESC 'All user ExtLogin for UserExtSource with Type IDP' EQUALITY caseIgnoreMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
 olcAttributeTypes: {60}( 1.3.6.1.4.1.8057.2.80.66 NAME 'schacPersonalUniqueCodes' DESC 'Unique codes of the user in Perun' EQUALITY caseIgnoreMatch SUBSTR caseIgnoreSubstringsMatch ORDERING caseIgnoreOrderingMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
 olcAttributeTypes: {61}( 1.3.6.1.4.1.8057.2.80.67 NAME 'rpIdentifier' DESC 'rpIdentifier of facility.' EQUALITY caseExactMatch SUBSTR caseExactSubstringsMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
+olcAttributeTypes: {62}( 1.3.6.1.4.1.8057.2.80.68 NAME 'rpEnvironment' DESC 'rpEnvironment' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 SINGLE-VALUE )
+olcAttributeTypes: {63}( 1.3.6.1.4.1.8057.2.80.69 NAME 'rpProtocol' DESC 'rpProtocol' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 SINGLE-VALUE )
```

Comment:
Also please don't forget to update description here as well.

Reply:
Done